src/main/java/edu/wright/airviewer2/AIRViewerController.java

```diff
@@ -182,37 +182,32 @@ public class AIRViewerController implements Initializable {
 
 	private AIRViewerModel promptLoadModel(String startPath) {
 
-		AIRViewerModel loadedModel = null;
-		try {
-			FileChooser fileChooser = new FileChooser();
-			fileChooser.setTitle("Open PDF File");
-			fileChooser.setInitialFileName(startPath);
-			Stage stage = null;
-			if(pagination.getScene() != null ) {
-			      stage = (Stage) pagination.getScene().getWindow();
-			}
-			File file = fileChooser.showOpenDialog(stage);
-			// modified the code to open only files with .pdf extension
-			if (null != file) {
-				path = file.getCanonicalPath();
-				while (!path.endsWith(".pdf")) {
-					System.out.println("select only pdf format files");
-					file = fileChooser.showOpenDialog(stage);
-					path = file.getCanonicalPath();
-				}
-				loadedModel = new AIRViewerModel(Paths.get(path));
+        AIRViewerModel loadedModel = null;
+        try {
+            FileChooser fileChooser = new FileChooser();
+            fileChooser.setTitle("Open PDF File");
+            fileChooser.setInitialFileName(startPath);
+            Stage stage = (Stage) pagination.getScene().getWindow();
```

Comment:
I see that this line of code is not synced with master. Can you please sync your changes with master